cmd/podman/system_prune.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podman/shared"
+	"github.com/containers/libpod/libpod/adapter"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var (
+	pruneSystemDescription = `
+	podman system prune
+
+        Remove unused data
+`
+	pruneSystemFlags = []cli.Flag{
+		cli.BoolFlag{
+			Name:  "all, a",
+			Usage: "remove all unused data",
+		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "Do not prompt for confirmation",
+		},
+		cli.BoolFlag{
+			Name:  "volumes",
+			Usage: "Prune volumes",
+		},
+	}
+	pruneSystemCommand = cli.Command{
+		Name:         "prune",
+		Usage:        "Remove unused data",
```

Comment:
"data" versus "systems"?

Reply:
Data is correct.  systems was incorrect.